build.gradle.kts

```diff
@@ -88,6 +88,7 @@ tasks.register<Zip>("releaseZip") {
 
 tasks.register<org.graalvm.build.samples.SamplesUpdateTask>("updateSamples") {
     inputDirectory.set(layout.projectDirectory.dir("samples"))
+    versions.put("nativeGradlePluginVersion", libs.versions.nativeGradlePlugin.get())
```

Comment:
Is there something preventing to use dot separators instead of camel case notation? That would be better for consistency.

Reply:
Dot separated properties cannot be used for versions in `plugins` group in `build.gradle`.
However as [Cédric pointed out](https://github.com/graalvm/native-build-tools/pull/113#discussion_r679803753) this is possible in `settings.gradle`, so I'll use that instead.